src/main/java/teammates/storage/entity/FeedbackResponse.java

```diff
@@ -35,6 +35,10 @@
 
     private FeedbackQuestionType feedbackQuestionType;
 
+    /**
+     * {@code giverEmail} does not necessarily contain an email. Depending on the question giver type,
+     * it may contain the giver's email, the team name, "anonymous", etc.
+     */
     private String giverEmail;
```

Comment:
We might want to change `giverEmail` to `giver` as well in the future, but maybe not now